### DISCUSSION TITLE
Meeting 15/10/21

### DISCUSSION THREAD

**Discussion by mathisfchx** [org-SCAN/website#156] [Meetings]
two sets of info, by two groups.
we need arrivals 
two blocks of data (emergency teams on people desembarcing, then the forward of the personns  : the accueuil)
they are doing the data validation, to be sure the data at the arival is the same that it was in the shelter
sample of cases, to look for a percentage of lookalike, the most sensitive : names are badly written + nationality
they come from a country at war, so comparaison country + name --> id
excell spread of spanish redcross, info exist for locations,
the people on shleter can be reinterogated,

interest more ::
data on people leaving, it's very fragmented data, different hands,
people that have all info in a group or not (very difficult)
they called the data people to cross names, 
==> it's means that what is desirable : data people to write directly on a server
the list is written by hand ::: better to enter directly on something : server
key informations : name and last name
important thing --> father's name and mother's name. (but bitrhday are kind of not good)
perhaps the region where they came from and where they embarqued
short info as posssible

mothers name father name origine
website like a form

form for migration events, in a team provide a means to link people to event,
list of the forms for events,

levenstein distance for names matches, when misspelling, "phase" lookup in excel (microsoft tool)

faudrait un outil ou les personnens qui partent puissent s'inscrire de n'importe ou avec un nombre exclusif à la croix rouge.

bracelet database is not the same on the people database
phonetic problem

issue nb 1 : shelter and departure differences,
issue nb 2 : curent, metereology (to imagine a likely position or that if the boat should have been arrived)

server on geneva definilty .

2FA auth of user connection for security. (sms auth)
each user a team to introduce data, icrc data. 

shorter form for the mobile app,
arrival site : much more data than the departure side.

CONCL:
Project 1) Data ingestion / presentation
Project 2) Correlation
Project 3) Travel Intelligence

**Comment by lduf** [comment]
<img width="526" alt="Capture d’écran 2021-10-15 à 12 19 09" src="https://user-images.githubusercontent.com/65276047/137598728-c631b1ef-6f85-4b26-a5ab-31fc1dd7fb12.png">
